### PR TITLE
 Settings and initial contents for Spanish localization

### PR DESCRIPTION
**Pull request by electrocucaracha**
This PR is to initiate Spanish localization based on

* Issue to #387 
* https://github.com/cncf/glossary/blob/main/LOCALIZATION.md guide

Base and target branch of this PR is dev-es which is development branch for Spanish localization.

This PR includes

- [languages.es] section in config.toml
- i18n/es.toml
- Some translated documents
  - Main page
    - content/es/_index.md
  - Others
    - contribute: content/es/contribute/_index.md
    - style-guide: content/es/style-guide/_index.md

### CODE REVIEW

**Comment by seokho-son** [comment]
@electrocucaracha 
Now Spanish Localization Team has https://github.com/cncf/glossary/tree/dev-es branch 
Please rebase and change the target branch to dev-es

**Comment by seokho-son** [comment]
@electrocucaracha 

I think this repository executes github/workflows/* (Github Actions) in `main` branch only.

Adding `.github/workflows/es-spellcheck.yml` to the `dev-es` branch only doesn't seem to work. (Am I right?)

In this PR, I'd like to focus on settings and initial contents for Spanish localization.
Can we handle Spell checker related files in another PR? (`.github/workflows/es-spellcheck.yml`, `.gitignore`)

**Review by krol3 — COMMENTED** [review]
LGTM

**Comment by seokho-son** [comment]
@electrocucaracha  ! Thank you for the contribution ! :)

We are going to check following tasks in this PR. (target branch is dev-es)

- [ ] 3-1. Adding a new language setting to site configuration https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-1-adding-a-new-language-setting-to-site-configuration
- [ ] 3-2. Adding a file for site strings https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-2-adding-a-file-for-site-strings
- [ ] 3-3. Adding a new localization directory https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-3-adding-a-new-localization-directory
- [ ] 3-4. Minimum required content https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-4-minimum-required-content
- [ ] 3-5. Check configuration works https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-5-check-configuration-works-and-open-a-pr

File: content/es/_index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Cualquier persona que desee hacer una contribución puede enviar un GitHub issue o crear un pull request. Asegúrese de seguir la [Guía de estilo](/style-guide/), leer el documento de [Cómo contribuir](/contribute/) y unirse al canal de #glossary en el CNCF Slack. También hay un canal #glossary-localizations para aquellos que quieran ayudar a traducir el glosario a su idioma.
+
+## Reconocimientos
+
+El Glosario Cloud Native fue iniciado por CNCF Marketing
+Committee (Business Value Subcommittee) e incluye
+contribuciones de [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) y muchos contribuidores mas. Para una lista de contribuidores completa, consulte [esta página de GitHub](https://github.com/cncf/glossary/graphs/contributors).
+
```
Comment (CathPag) [inline content/es/_index.md:21]: Please add a paragraph here mentioning the localization team. Something like:

 "The Spanish Glossary version includes contributions from...[add the names of main contributors]"

**Review by CathPag — COMMENTED** [review]
Looks great! Just need to add the localization team :)

**Review by CathPag — APPROVED** [review]
Looks great!

**Pull request merged**